apps/evm/src/clients/api/queries/useGetPool/__snapshots__/index.spec.tsx.snap

```diff
@@ -5,6 +5,7 @@ exports[`useGetPool > returns the correct asset 1`] = `
   "pool": {
     "assets": [
       {
+        "badDebtMantissa": 0n,
```

Comment:
Just to cover this new feature with tests, it would be good to have some test assets with a non-zero bad debt.